### PR TITLE
docs: fix typos

### PR DESCRIPTION
**Pull request by ben46**
# Docs PR Template

## Why this should be merged

These changes are necessary because they fix typos in the documentation. Correcting typos ensures clarity and professionalism in the documentation, improving the overall user experience and understanding of the content.

## How this works

The changes involve correcting typographical errors found in the documentation files. This includes fixing spelling mistakes, grammatical errors, and punctuation inconsistencies. The corrections are made directly within the affected markdown files.

## How these changes were tested

To view the changes, one can review the affected markdown files either locally or through the version control system. The changes were tested by running the following checks:

1. The `vale` command was executed on all changed `.md` files to ensure compliance with grammar rules.
2. The `markdownlint` command was executed on all changed `.md` files to ensure adherence to linting rules.
3. Additional checks outlined in `style-checker-notes.md` were completed to ensure comprehensive validation of the changes.

### If a document was removed/deleted

- Ensure that the path to the deleted document is redirected to a valid URL via the `_redirects` file.
- Verify the `_redirects` file manually using the cloudflare preview link to ensure correctness.
- Update `sidebars.json` to reflect any changes made to file paths.

### If a document was moved

- Redirect the paths of all files that were moved to a new location via the `_redirects` file.
- Manually verify the `_redirects` file using the cloudflare preview link to confirm accuracy.
- Update `sidebars.json` to accurately reflect any changes made to file paths.

### CODE REVIEW

**Comment by laviniat1996** [comment]
Thank you for your contribution! I've left a small comment

**Comment by laviniat1996** [comment]
@ben46 Hi, can you please address my comment so I can merge this?

**Comment by ben46** [comment]
> @ben46 Hi, can you please address my comment so I can merge this?

sorry, I am rookie on using pull request, I can not find your comment.

File: docs/build/subnet/deploy/multisig-auth.md [hunk 1]
```diff
@@ -376,17 +376,17 @@ Your subnet auth keys for add validator TX creation: [P-avax1kdzq569g2c9urm9887c
 ```
 
 ### Finish Assembling the TX
```
Comment (meaghanfitzgerald) [inline docs/build/subnet/deploy/multisig-auth.md:378]:
```suggestion
### Finish Assembling the TX

```

**Review by dubrado — DISMISSED** [review]

**Comment by meaghanfitzgerald** [comment]
@laviniat1996 please follow up on this

**Comment by laviniat1996** [comment]
/gt docs/build/dapp/smart-contracts/verification/verify-hardhat.md i18n/es/docusaurus-plugin-content-docs/current/build/dapp/smart-contracts/verification/verify-hardhat.md spanish

**Comment by laviniat1996** [comment]
Replaced by #1751

**Pull request closed without merging**